recipes/rgccacmd/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "rgcca" %}
+{% set version = "3.0.0" %}
```

Comment:
Is there a reason that 3.0.0 isn't on CRAN?

Reply:
thank you for the bug fix. the next version of the package won't be ready for at least several months and I'm finishing my job soon. There is nobody in the team with conda (and galaxy) skills, so we wanted to validate my tool with a pre-release before I leave, otherwise it won't be (and especially for galaxy requirements). in a second step, I could update the conda with the CRAN version when it will be released because the cost will be minimal.